src/components/form-fields/combobox/combobox-form-field.tsx

```diff
@@ -29,18 +29,21 @@ export function ComboboxFormField<TFormModel extends FieldValues, TFieldModel>({
     <FormField
       control={control}
       name={name}
-      render={({ field }) => (
-        <ComboboxFormItem
-          label={label}
-          description={description}
-          required={required}
-          buttonClassName={buttonClassName}
-          onSelect={field.onChange}
-          fieldValue={field.value}
-          inForm={true}
-          {...props}
-        />
-      )}
+      render={({ field }) => {
+        const _fieldValue = props.initialValues ?? field.value;
```

Comment:
Refactor after https://alcosi.atlassian.net/browse/TWINFACES-207

Reply:
Let's fix this issue in a [separate task](https://alcosi.atlassian.net/browse/TWINFACES-213).